plugins/plugin-zend-debugger/che-plugin-zend-debugger-server/src/main/java/org/eclipse/che/plugin/zdb/server/ZendDbgLocationHandler.java

```diff
@@ -26,12 +26,12 @@
 
   public static final Location createVFS(
       String target, String resourceProjectPath, int lineNumber) {
-    return new LocationImpl(target, lineNumber, false, 0, resourceProjectPath, null, -1);
+    return new LocationImpl(target, lineNumber, false, "", resourceProjectPath, null, -1);

```

Comment:
The fact hat we have to change somthing in the php debugger is a code smell in itself. But again...not right now.